plugin/pki/path_venafi_cert_enroll.go

```diff
@@ -113,6 +134,13 @@ func (b *backend) pathVenafiSign(ctx context.Context, req *logical.Request, data
 		return logical.ErrorResponse(fmt.Sprintf("unknown role: %s", roleName)), nil
 	}
 
+	ttl := time.Duration(data.Get("ttl").(int)) * time.Second
+	if role.MaxTTL > 0 && ttl > role.MaxTTL {
+
+		return logical.ErrorResponse(errorTextValueMustBeLess), nil
```

Comment:
Same comment as above.  The validity of the Venafi enrolled certificate should be the role `max_ttl` if this situation exists.

Reply:
fixed on the latest commit